tests/e2e/tests/base.py

```diff
@@ -48,48 +47,66 @@ def get_expected_fx_pkg_str(self, os, alias, product_version):
                 (os, alias, product_version)
             raise ValueError(e)
 
-    def _head_request(self, url, user_agent=_user_agent_firefox,
-                      locale='en-US', params=None):
+    def verify_redirect_to_correct_product(self, base_url, fx_pkg_name, get_params):
+        """Given a set of GET params, this method verifies bouncer redirects to
+        the expected Firefox product.
+
+        :arg base_url: The server under test.
+        :arg fx_pkg_name: The full Firefox package name that is the expected download.
+        :arg get_params: The GET params to pass to bouncer.
+        """
+        response = self.request_with_headers(base_url, params=get_params)
+        parsed_url = urlparse(response.url)
+        # verify service is up and a 200 OK is returned
+        assert requests.codes.ok == response.status_code, \
```

Comment:
We're currently running our tests with `pytest`, but you raise an interesting point. I'll have to think about this.